server/tserver/src/main/java/org/apache/accumulo/tserver/log/DfsLogger.java

```diff
@@ -433,11 +427,11 @@ public synchronized void open(String address) throws IOException {
       byte[] cryptoParams = encrypter.getDecryptionParameters();
       CryptoUtils.writeParams(cryptoParams, logFile);
 
-      /**
-       * Always wrap the WAL in a NoFlushOutputStream to prevent extra flushing to HDFS. The
-       * {@link #write(LogFileKey, LogFileValue)} method will flush crypto data or do nothing when
-       * crypto is not enabled.
-       **/
+      /*
+       * Always wrap the WAL in a NoFlushOutputStream to prevent extra flushing to HDFS. The {@link
+       * #write(LogFileKey, LogFileValue)} method will flush crypto data or do nothing when crypto
```

Comment:
The `@link` is a javadoc tag. It doesn't do anything in regular comments. This wasn't a valid javadoc in the first place, inside the method, so that change was fine, but it highlights that this javadoc tag can go away too.

Reply:
updated comment in c543b2e50f